### PR TITLE
Better Simplified Chinese translations

### PR DESCRIPTION
**Pull request by liu-yun**
Since the old PR was closed, I have to create a new one.

### CODE REVIEW

**Comment by M66B** [comment]
Since there are a lot of changes, I like to see this reviewed by someone else.

**Comment by darkgentry** [comment]
@wongsyrone @liveasx @liudongmiao Plz review.

File: res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -1,201 +1,145 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
 
-    <string name="app_notenabled">XPrivacy 没有在 Xposed 框架中启用</string>
+    <string name="app_notenabled">XPrivacy 没有在 Xposed Installer 中启用</string>
```
Comment (wongsyrone) [inline res/values-zh-rCN/strings.xml]: Better not change this line

File: res/values-zh-rCN/strings.xml [hunk 2]
```diff
@@ -1,201 +1,145 @@
...
     <string name="menu_settings">设置</string>
-    <string name="menu_template">新装应用默认权限模板</string>
+    <string name="menu_template">权限模板</string>
     <string name="menu_report">报告问题</string>
-    <string name="menu_export">导出全局配置</string>
-    <string name="menu_import">导入全局配置</string>
-    <string name="menu_submit">提交权限限制规则</string>
-    <string name="menu_fetch">下载权限限制规则</string>
-    <string name="menu_theme">切换明/暗主题</string>
+    <string name="menu_export">导出</string>
```
Comment (wongsyrone) [inline res/values-zh-rCN/strings.xml]: Line 16-20 seems no need to change cause current ones are clear enough.

**Comment by wongsyrone** [comment]
Why there are only 200 lines in your file? You'd better check it again, and the origin locates here --> res/values/strings.xml.

File: res/values-zh-rCN/strings.xml [hunk 3]
```diff
@@ -1,201 +1,145 @@
...
-    <string name="title_template_full">应用模板(分类和功能)</string>
-    <string name="title_template_merge">应用模板(合并)</string>
-    <string name="title_template_merge_reset">应用模板 (合并重置)</string>
-    <string name="title_enable_ondemand">开启按需权限限制</string>
-    <string name="title_disable_ondemand">关闭按需权限限制</string>
+    <string name="title_set_restrict">限制 (分类)</string>
+    <string name="title_template_category">应用模板 (类别)</string>
+    <string name="title_template_full">应用模板 (类别和函数)</string>
+    <string name="title_template_merge">应用模板 (合并原有)</string>
+    <string name="title_template_merge_reset">应用模板 (清除原有)</string>
```
Comment (darkgentry) [inline res/values-zh-rCN/strings.xml]: Line120 : should be 应用模板 (清除模板已设置的项)

**Comment by liu-yun** [comment]
@wongsyrone Deleted the strings tagged as tools:ignore="UnusedResources".

**Pull request closed without merging**